web/adelacecilia/html/datalist tag/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>The "datalist" tag</h1>
```

Comment:
Please dont leave empty lines in your HTML :)

Reply:
please delete blank line 4.

Reply:
Sorted! 

Thank you!